polyfills/fm/client/fmradio.js

```diff
@@ -0,0 +1,553 @@
+// navigator.mozFMRadio polyfill!
+// https://developer.mozilla.org/es/docs/Web/API/Navigator/mozFMRadio
+
+// I have to define the service protocol, and this is as good place as any. The
+// protocol will be as follows:
+// * Each request will have a numeric request id, set by the client
+// * The answers will include the request id, so the client can know to which
+//   request the answer corresponds
+// * The request id will also act as identifier for remote objects (locks and
+//  observers!)
+
+// For all the get|set operations:
+//  * Request: { id: requestId,
+//               data: {
+//                 operation: 'set'|'get',
+//                 name: propertyName,
+//                 value: propertyValue (set request only)
+//               }
+//             }
+// * Answer: { id: requestId,
+//             result|error: Whatever }
+
+// For all the invoke operations:
+//  * Request: { id: requestId,
+//               data: {
+//                 operation: 'invoke',
```

Comment:
Why do we need two attributes here? Why don't we just use operation: methodname?